
    Tomasz> Forgotten / somewhere? 

    Robert> Hmm. Any chance of catching that in gdb so we can see where itÊ¼s being
    Robert> called from?

Completely WAG:

---
 src/xrdb.c  |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- src/xrdb.c
+++ src/xrdb.c	2020-08-15 06:15:14.059639732 +0000
@@ -292,9 +292,9 @@ get_user_app (const char *class)
       /* Check in the home directory.  This is a bit of a hack; let's
 	 hope one's home directory doesn't contain ':' or '%'.  */
       char const *home = get_homedir ();
-      db = search_magic_path (home, class, "%L/%N");
+      db = search_magic_path (home, class, "/%L/%N");
       if (! db)
-	db = search_magic_path (home, class, "%N");
+	db = search_magic_path (home, class, "/%N");
     }
 
   return db;
